### PR TITLE
Update issue templates

### PR DESCRIPTION
**Pull request by edsonbrusque**

### CODE REVIEW

**Comment by johnnyasantoss** [comment]
Nice! 👍

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots**
+If applicable, add screenshots to help explain your problem.
+
+**Desktop (please complete the following information):**
```
Comment (johnnyasantoss) [inline .github/ISSUE_TEMPLATE/bug_report.md:26]: I think that this desktop section and the browser info are not needed

**Review by johnnyasantoss — CHANGES_REQUESTED** [review]
Made a comment on the bug report. Besides that LGTM!

File: .github/ISSUE_TEMPLATE/bug_report.md (hunk 1 above)
Comment (edsonbrusque) [reply .github/ISSUE_TEMPLATE/bug_report.md:26]: Right.

Comment (edsonbrusque) [reply .github/ISSUE_TEMPLATE/bug_report.md:26]: Deleted the browser lines.
I would prefer to maintain the Desktop section as we'll probably have a desktop version LibrePay. And it doesn't makes a difference while the app is Android/iOS only.

Comment (johnnyasantoss) [reply .github/ISSUE_TEMPLATE/bug_report.md:26]: Yeah, that is okay :)

**Review by johnnyasantoss — APPROVED** [review]

**Pull request merged**